generators/css-framework/modules/bootstrap/main.scss

```diff
@@ -54,9 +54,9 @@ footer {
   border-top: 1px solid #e0e0e0;
 }
 
+// Solves issue #106 https://github.com/sahat/megaboilerplate/issues/106
```

Comment:
Looks good, thanks. Just one minor thing, could you remove these "Solves issue" comments?